github/ipfs-shipyard.yml

```diff
@@ -4599,29 +4499,16 @@ teams:
     description: People with write access to Javascript repositories
     members:
       maintainer:
-        - 2color
         - lidel
-        - rvagg
-        - sgtpooki
```

Comment:
Im not a fan of being removed here. Will I still have write permissions?

Reply:
As per https://github.com/ipfs-shipyard/github-mgmt/pull/77#issuecomment-1963562925, here's your entire access changeset:

```
User SgtPooki:
  - will lose push permission to ipfs-thing-2023
```

This suggests to me `Javascript` team is not used much after all.

Reply:
Nah its probably fine